packages/dart/npt_flutter/lib/features/profile/bloc/profile_bloc.dart

```diff
@@ -114,12 +114,14 @@ class ProfileBloc extends LoggingBloc<ProfileEvent, ProfileState> {
     // ProfileLoaded and ProfileFailedSave are both ProfileLoadedState
     var profile = (state as ProfileLoadedState).profile;
     emit(ProfileStarting(uuid, profile: profile));
+    App.navState.currentContext?.read<ProfilesRunningCubit>().prepare(uuid);
```

Comment:
Note for future self: prepare adds the `<uuid>: null` entry to the profiles running map. Thus profiles which should be readonly can be calculated in bulk, if it's got a uuid in the key entry of the map it should be readonly, it's got a socketConnector in the value entry of the map it started successfully.